cmd/crc/cmd/start.go

```diff
@@ -133,3 +137,19 @@ func getPullSecretFileContent() (string, error) {
 
 	return pullsecret, nil
 }
+
+func checkIfNewVersionAvailable(noUpdateCheck bool) {
+	if noUpdateCheck {
+		return
+	}
+	newVersionAvailable, newVersion, err := version.NewVersionAvailable()
```

Comment:
`isNewVersionAvailable`, else it is too close to the variable named `newVersion` which likely contains a string.

Reply:
This is later used in the if statement and `if newVersionAvailable` sounds more natural than `if isNewVersionAvailable`.